src/Atmos/Model/atmos_tendencies.jl

```diff
@@ -13,6 +13,7 @@ filter_source(pv::PV, s::Gravity{PV}) where {PV <: Momentum} = s
 filter_source(pv::PV, s::GeostrophicForcing{PV}) where {PV <: Momentum} = s
 filter_source(pv::PV, s::Coriolis{PV}) where {PV <: Momentum} = s
 filter_source(pv::PV, s::RayleighSponge{PV}) where {PV <: Momentum} = s
+filter_source(pv::PV, s::EnergySponge{PV}) where {PV <: Energy} = s
```

Comment:
It looks like we need to rebase this branch/PR. There is now a second argument in this method (the balance law)